src/details/ArborX_DetailsHeap.hpp

```diff
@@ -114,6 +123,14 @@ KOKKOS_INLINE_FUNCTION void sortHeap(RandomIterator first, RandomIterator last,
     popHeap(first, last--, comp);
 }
 
+template <typename RandomIterator, typename Compare>
+KOKKOS_INLINE_FUNCTION void heapSort(RandomIterator first, RandomIterator last,
+                                     Compare comp)
```

Comment:
Drop it

Reply:
No. It's reasonable to have a function that sorts an array.

Reply:
It is reasonable not to introduce confusion to avoid calling two familiar functions.
Blocking.

Reply:
You can introduce a `makeHeapAndSortIt` in the DBSCAN header if you really want it.  It does not belong here.

Reply:
Fixed.